### DISCUSSION TITLE
加入分栏对和生词功能

### DISCUSSION THREAD

**Discussion by guevara** [immersive-translate/immersive-translate#1196] [Ideas]
### 新功能 | New Feature

1. 分栏功能, 两侧分别为原文和译文, 滚动或者翻页时, 原文和译文同步
2. 自动高亮和标记生词, 用颜色区分蓝思值, 支持像relingo一样忽略和添加生词, 并在面板上直接直接当前页生词列表
3. 生词可以背诵和导出, 功能参考anki和不背单词

### 为什么您想要添加这些功能 | Why you want to add these features

1. 当前译文显示在原文下方, 原文和译文交错在一起, 用户的关注焦点容易完全被译文带走, 对于英语学习者不太友好. 显示器多为宽屏幕, 增加分栏对比, 可以有意识的将用户的焦点从原文拉开, 或者直接查看和下载全部译文
2. 用户在使用翻译的最终目的其实是为了减少对翻译工具的直接依赖. 通过自动高亮和标记生词, 可以帮助透析学习英语
3. 提取/背诵生词功能有利于增加产品竞争力

**Comment by theowenyoung** [comment]
感谢建议，但是目前沉浸式翻译的主要目标是要保持简单，刻意不加入过多的功能。

这是一个很好的产品点子～ 所以也许以后有精力的话，可以独立做一个新的插件～
